ion/src/simulator/android/src/java/io/github/omega/simulator/OmegaActivity.java

```diff
@@ -39,4 +39,19 @@ public Bitmap retrieveBitmapAsset(String identifier) {
     }
     return bitmap;
   }
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    /* This is done to hide the status bar and the bottom navigation buttons.
+     *
+     * In SDLActivity::onCreate, setWindowStyle(false) is called, which means
+     * the fullscreen mode is put to false. We call again the method here with
+     * true, in order not to modify the external sources.
+     *
+     * TODO: This was not needed for v12 of Epsilon, even though
+     * setWindowStyle(false) was already called in SDLActivity::onCreate. Find
+     * out why and make a proper fix? */
+    super.onCreate(savedInstanceState);
+    setWindowStyle(true);
+  }
```

Comment:
This is a nice addition, but has nothing to do in this PR. Please open another one.

Reply:
I've committed this accidentally, I will change that soon